src/nutrients_plankton_bacteria_detritus.jl

```diff
@@ -0,0 +1,131 @@
+using Oceananigans.Units: day
+using Oceananigans.Grids: znode, Center
+using Oceananigans.Fields: ZeroField, ConstantField
+using Oceananigans.Biogeochemistry: AbstractBiogeochemistry
+
+import Oceananigans.Biogeochemistry: required_biogeochemical_tracers, biogeochemical_drift_velocity
+
+const c = Center()
+
+"""
+    NutrientsPlanktonBacteriaDetritus(; kw...)
+
+Return a four-tracer biogeochemistry model for the interaction of nutrients, plankton
+bacteria, and detritus.
+
+Parameters
+==========
+
```

Comment:
We still need to fill these out -- I'm not entirely sure how to describe the `bacteria_yield` (otherwise I could finish this).